app/src/main/java/com/pysiak/simple/clock/ui/ClockPresenter.java

```diff
@@ -0,0 +1,47 @@
+package com.pysiak.simple.clock.ui;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import com.pysiak.simple.clock.R;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * Class for implementing the architecture of the MVP
```

Comment:
Make javadoc in next styling: Description in one line (if possible), one extra line between description and @param/@author/@return

Reply:
Add new line after Decription